src/js/stores/MetronomeStore.js

```diff
@@ -166,10 +166,40 @@ class MetronomeStore extends EventEmitter {
     MetronomeActions.runJob(jobID);
   }
 
-  suspendSchedule(jobID) {
-    let schedule = this.getJob(jobID).schedules[0];
-    schedule.enabled = false;
-    MetronomeActions.suspendSchedule(jobID, schedule);
+  disableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
+
+      return null;
+    }
+
+    let [schedule] = job.getSchedules();
+    if (schedule == null) {
+
+      return null;
+    }
+
+    this.updateSchedule(jobID, Object.assign({}, schedule, {enabled:false}));
+  }
+
+  enableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
+
+      return null;
+    }
+
+    let [schedule] = job.getSchedules();
+    if (schedule == null) {
+
+      return null;
+    }
+
+    this.updateSchedule(jobID, Object.assign({}, schedule, {enabled:true}));
+  }
+
+  updateSchedule(jobID, schedule) {
```

Comment:
That wouldn't work as schedule would be defined as `{enabled:true}` if we don't do the respective checks. But we should also check for `null` values here.